File: baseble/src/main/java/com/fcbox/ble/baseble/callback/IConnectCallback.java
package com.fcbox.ble.baseble.callback;


import com.fcbox.ble.baseble.core.DeviceMirror;
import com.fcbox.ble.baseble.exception.BleException;


/**
 * 连接设备回调
 */
public interface IConnectCallback {
    //连接成功
    void onConnectSuccess(DeviceMirror deviceMirror);

    //连接失败
    void onConnectFailure(BleException exception);

    //连接断开
    void onDisconnect(boolean isActive);
}

File: baseble/src/main/java/com/xzy/ble/baseble/callback/IConnectCallback.kt
package com.xzy.ble.baseble.callback
import com.xzy.ble.baseble.core.DeviceMirror
import com.xzy.ble.baseble.exception.BleException


/**
 * 连接设备回调
 */
interface IConnectCallback {
    //连接成功
    fun onConnectSuccess(deviceMirror: DeviceMirror)

    //连接失败
    fun onConnectFailure(exception: BleException)

    //连接断开
    fun onDisconnect(isActive: Boolean)
}
